### DISCUSSION TITLE
Deepgram's Speech-to-Text  misses complete sentences sometimes, need help.

### DISCUSSION THREAD

**Discussion by umarmirza0341** [deepgram/community#887] [General help]
We are currently working on a proof of concept and have selected the Deepgram Nova2 model for transcribing our MP3 audio files. We are testing the model's accuracy and have found that Deepgram's Speech-to-Text Nova2 model provides good results overall. However, we have noticed that it occasionally misses complete sentences. Does anyone know the reason for this?
Options we used are as follows:

options = PrerecordedOptions(
                smart_format=True, model="nova-2", language="en-US", diarize=True
            )
response = deepgram.listen.prerecorded.v('1').transcribe_file(payload, options, timeout=3000)

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by davidvonthenen** [comment]
Hi @umarmirza0341 

Since you are talking about pre-recorded audio, do you have some sample audio of where this is happening? It might be easier to take a look at an example of this happening as a point of discussion.
